frontend/src/components/Chatbox.tsx

```diff
@@ -21,10 +63,14 @@ const Chatbox = () => {
 
     // Send the input to the FastAPI backend
     try {
-      const response = await apiService.sendQuery(input);
+      //Console.log(sessionId);
+      if (sessionId === null) {throw new Error('sessionId was null!');}
+
+      const response = await apiService.sendQuery(input, sessionId);
     
       responseMessage.text = JSON.stringify(response.data);
     } catch {
+      //Console.log(e);
       responseMessage.text = 'Error: Something went wrong';
```

Comment:
Maybe actually catch the error and show meaningful message? Couple of lines above you throw "sessionId was null", why not print that?

As far as I know our errors areonly when `sessionId` is null or `JSON.stringify` fails (although I don't think JSON.stringify can fail*). So we wouldn't even need the try catch here (unless `sendQuery` can fail, but still async/await functions behave weirdly in throwing).

Please check what can throw and maybe provide more meaningful message, or simplify it ;)

Reply:
I initially tried to console.log the error, but the linter doesn't allow it